tests/tt_metal/tt_fabric/fabric_data_movement/test_basic_1d_fabric.cpp

```diff
@@ -1751,5 +1751,412 @@ TEST_F(Fabric1DFixture, DISABLED_TestEDMConnectionStressTestQuick) {
     }
 }
 
+void UnicastCommon(BaseFabricFixture* fixture, NocSendType noc_send_type) {
```

Comment:
Super small nit for readability - similar suggestion for multicast:
```suggestion
void FabricUnicastCommon(BaseFabricFixture* fixture, NocSendType noc_send_type) {
```